src/Microsoft.ML.Transforms/Text/LdaTransform.cs

```diff
@@ -936,7 +938,56 @@ private protected override IRowMapper MakeRowMapper(DataViewSchema schema)
             => new Mapper(this, schema);
     }
 
-    /// <include file='doc.xml' path='doc/members/member[@name="LightLDA"]/*' />
+    /// <summary>
+    /// The LDA transform implements <a href="https://arxiv.org/abs/1412.1576">LightLDA</a>, a state-of-the-art implementation of Latent Dirichlet Allocation.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes |
+    /// | Input column data type | [key](xref:Microsoft.ML.Data.KeyDataViewType) data types|
```

Comment:
> data types [](start = 80, length = 11)

just 'key type'

Reply:
Why not just use only xref? That would make references to Key consistent.

---
In reply to: [277098008](https://github.com/dotnet/machinelearning/pull/3442#discussion_r277098008) [](ancestors = 277098008)